### DISCUSSION TITLE
Add new scenario showing how to display tags - all the steps

### DISCUSSION THREAD

**Discussion by sympmarc** [microsoft-search/pnp-modern-search#3848] [Ideas]
It would be helpful to have a scenario showing all the steps to display tags well. @kasperbolarsen has a fix for the tag display which will make showing the tags even more appealing.

Please assign this to me.

**Comment by sympmarc** [comment]
@kasperbolarsen and/or @wobba - In reading through the Slots doc, it says you should map the `ows_taxId_*` Crawled Property to the Managed Property. I *never* do this, mapping the `ows_*` Crawled Property (no `_taxId`) instead. This works for me consistently, but I'm never using more than one language. Is the `ows_taxId_*` mapping required for multi-lingual content, i.e., for true "localization"?

**Comment by wobba** [comment]
@sympmarc mapping ows_foo for a taxonomy property will by default have the label - unless you change the setting to what is displayed for the column. Using the taxid version has the guids as part of it, which is needed to look up and translate a term. If you only have one language, then sure you can use the label name as that would typically be unique.

Having the guid also ensure any renamed label is shown correctly.

**Comment by sympmarc** [comment]
Thanks for the clarification, @wobba. I shouldn't be so English-speaking in my usage.
